include/ck/host_utility/kernel_launch.hpp

```diff
@@ -156,3 +156,82 @@ float launch_and_time_kernel_with_preprocess(const StreamConfig& stream_config,
     return 0;
 #endif
 }
+
+// return time that does not include preprocess's time
+template <typename... Args, typename F, typename PreProcessFunc>
+float launch_and_time_kernel_without_preprocess(const StreamConfig& stream_config,
```

Comment:
This function name looks confusing, can we change it to:
launch_and_time_kernel_flush_cache

Reply:
This function is not only for flush cache, all call preprocess that does not calculate time  can use this function